src/components/Navbar/Navbar.tsx

```diff
@@ -15,6 +16,7 @@ const Navbar = () => {
         <Logo>
           weather<span>ly</span>
         </Logo>
+        <CurrentLocationWidget />
```

Comment:
As written in the description of the story, this information shouldn't be shown on the main page